pkg/controller/ttl_manager_test.go

```diff
@@ -203,6 +203,86 @@ func TestResolveTtlStatus(t *testing.T) {
 			expectedErr:   true,
 			expectedRetry: true,
 		},
+		{
+			name: "last transition before the activeAnnotation",
```

Comment:
please say why _and_ what in these names -- see my tests above. this one could be:

> last transition is before the activeAnnotation *so it is ignored*